### DISCUSSION TITLE
Cannot wake from sleep

### DISCUSSION THREAD

**Discussion by ItsDan001** [MultimediaLucario/Lenovo-ThinkPad-T440s#26] [Q&A] [answered]
I am running macOS Monterey, with an intel i5-4300U, and my issue is pretty much after the laptop goes to sleep, it cannot wake up after. Cannot wake it up at all. tried with the keyboard, trackback, heck even the power button doesn't wake it up. So, will someone be able to help me? (And no, disabling sleep is not a solution for me = / )

**Comment by MultimediaLucario** [comment]
Just hit the Fn key to wake the computer

**Reply by ItsDan001** [reply]
That does not work either

**Reply by MultimediaLucario** [reply]
Open the Config.plist file go to ACPI, expand the Patch section, open the Item 27 and set the instant wake fix enabled toggle to yes.

**Comment by MultimediaLucario** [comment, marked as answer]
Open the Config.plist file go to ACPI, expand the Patch section, open the Item 27 and set the instant wake fix enabled toggle to yes.
